pandas/core/frame.py

```diff
@@ -2201,10 +2201,21 @@ def to_feather(self, path, **kwargs) -> None:
     @Substitution(klass="DataFrame")
     @Appender(_shared_docs["to_markdown"])
     def to_markdown(
-        self, buf: Optional[IO[str]] = None, mode: Optional[str] = None, **kwargs
+        self,
+        buf: Optional[IO[str]] = None,
+        mode: Optional[str] = None,
+        index: Optional[bool] = True,
```

Comment:
this is technically a breaking change i think, but how much do we care, cc @TomAugspurger

Reply:
I think just change the ValueError to a FutureWarning and we're fine, right?